### PR TITLE
Add 5.3 ld

### PR DESCRIPTION
**Pull request by simonlindholm**
I haven't checked if this breaks anything else, it very well might.

### CODE REVIEW

File: libc_impl.c [hunk 1]
```diff
@@ -2301,12 +2301,22 @@ uint32_t wrapper_setlocale(uint8_t *mem, int category, uint32_t locale_addr) {
 }
 
 uint32_t wrapper_mmap(uint8_t *mem, uint32_t addr, uint32_t length, int prot, int flags, int fd, int offset) {
+    if (addr == 0 && prot == (prot & 3) && flags == 2) {
+        // Read/write, map anonymous. Just make a copy.
+        uint8_t *ptr = mmap(0, length, prot, flags, fd, offset);
```
Comment (AngheloAlf) [inline libc_impl.c:2306]: so this isn't a problem on windows?

File: libc_impl.c [hunk 2]
```diff
@@ -2673,8 +2683,35 @@ uint32_t wrapper_tfind(uint8_t *mem, uint32_t key_addr, uint32_t rootp_addr, uin
...
-    assert(0 && "qsort not implemented");
+    // We don't let recompiled programs run multithreaded, and hopefully
+    // there's no re-entrancy, so using a global should be fine...
+    global_qsort_meta = (struct QsortMeta) {
+        .mem = mem,
+        .trampoline = trampoline,
+        .compare_addr = compare_addr,
+        .sp = sp,
+    };
+    qsort((void*)(uintptr_t)base_addr, num, size, qsort_comp);
```
Comment (AngheloAlf) [inline libc_impl.c:2714]: yikes
What about `qsort_s`? That function allows to pass a context parameter https://en.cppreference.com/w/c/algorithm/qsort

**Review by hensldm — APPROVED** [review]
LGTM, tested linux builds and they seem fine.

**Review by EllipticEllipsis — APPROVED** [review]
Fine on Mac.

**Pull request merged**